src/components/Content/ExperimentsContent/Experiment/ExperimentHeader/_/Container.js

```diff
@@ -58,27 +56,29 @@ const ExperimentHeaderContainer = ({
   trainingLoading,
   deleteTrainingLoading,
   handleRemoveOperator,
-  handleFetchExperiment,
   handleEditExperimentName,
-  handleTrainExperiment,
-  handleDeleteTrainExperiment,
+  handleFetchOperators,
+  handleCreateExperimentRun,
+  handleDeleteExperimentRun,
 }) => {
   const { projectId, experimentId } = useParams();
 
   // HOOKS
   // did mount hook
   useEffect(() => {
     if (experimentId) {
-      handleFetchExperiment(projectId, experimentId);
+      handleFetchOperators(projectId, experimentId);
+      experiment(experimentId);
```

Comment:
@miguelfferraz 
Line 73:6:  React Hook useEffect has a missing dependency: 'experiment'. Either include it or remove the dependency array. If 'experiment' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps